core/lib/presentation/views/html_viewer/html_content_viewer_widget.dart

```diff
@@ -95,6 +95,7 @@ class _HtmlContentViewState extends State<HtmlContentViewer> {
       backgroundColor: Colors.white,
       onWebViewCreated: (controller) async {
         _webViewController = controller;
+       _htmlData = _htmlData?.replaceFirst('overflow: hidden', 'overflow: auto');
```

Comment:
- I don't think it solves this problem.
- You should note that if you fix this, some emails may be displayed with 2 scrollbars while reading emails on mobile. You should retry the view cases.
- Please show us the demo video to see the results more clearly.

Reply:
I added at top comment. Please check it